contract-ui/tabs/claim-conditions/components/claim-conditions-form/index.tsx

```diff
@@ -267,15 +423,22 @@ export const ClaimConditionsForm: React.FC<ClaimConditionsFormProps> = ({
 
       <Flex onSubmit={handleFormSubmit} direction="column" as="form" gap={10}>
         <Flex direction={"column"} gap={6}>
+          {/* Show the reason why the form is disabled */}
+          {!isAdmin && (
+            <Text>Connect with admin wallet to edit claim conditions.</Text>
+          )}
           {controlledFields.map((field, index) => {
```

Comment:
pull the mapped function out as a component and add a boolean state `isExpanded` to keep track of whether the phase should be expanded or collapsed instead of manually keeping track of it using `editingPhases`